openapi/components/paths/files.yaml

```diff
@@ -288,7 +288,7 @@ paths:
                   format: binary
                   description: The binary blob of the png file.
                 tag:
-                  type: string
```

Comment:
I wouldn't consider this the same tag, it is an explicit union of:
* icon, gallery, sticker, emoji, or emojianimated

Reply:
Fair, that one makes sense. I don't think that one is one of the ones that gave errors now that I look at it again. Feel free to not include that one.